test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,28 @@
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
This LO shouldn't be part of this PR. In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the master branch that is in sync with the upstream master branch